README.md

```diff
@@ -1,2 +1,166 @@
-# React-Capstone
-This React capstone project is about building a mobile web application to check a list of metrics (numeric values) that I will create making use of React and Redux.
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 React Capstone - CoinGecko <a name="about-project"></a>
+
+This React capstone project is about building a mobile web application to check a list of metrics (numeric values) that I will create making use of React and Redux. It's connected to CoinGecko API to get several data about cryptocurrencies.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.hostinger.com/tutorials/what-is-react">REACT</a></li>
+    <li><a href="https://redux-toolkit.js.org/">REDUX TOOLKIT</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Follow a correct Gitflow**
+- **Comply with linters**
+- **Use React components**
+- **Use React Router**
+- **Understand the concept of state management**
+- **Use store, actions, and reducers in React**
+- **Setup a Redux store**
+- **Create a slice to save state and reducers**
+- **Connect React and Redux**
+- **Connect an existing API via Redux and use the data to display something in a UI**
+- **Apply styling**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live site](https://coingecko-4xi2.onrender.com/metrics)
+- [Loom video](https://www.loom.com/share/334c5aca12bc46f48e7bd8a587e0dc19)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps:
+
+### Prerequisites
+
+In order to run this project you need:
+
+✅ Github account <br>
+✅ Visual Studio Code installed <br>
+✅ Node.js installed <br>
+✅ Git Bash installed (optional)   
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd [my-folder]
+  git clone "putHereRepoLink"
+```
```

Comment:
- [ ] Kindly add your project's repo link here so it's easily accessible and not placeholder 👍🏽

Reply:
Ok, done it